docs/website/docs/dlt-ecosystem/destinations/filesystem.md

```diff
@@ -181,13 +181,14 @@ bucket_url = "abfss://<container_name>@<storage_account_name>.dfs.core.windows.n
 
 You can use `az`, `abfss`, `azure` and `abfs` url schemes.
 
-If you need to use a custom host for your account you can set it up like below:
+If you need to use a custom host to account your storage account you can set it up like below:
```

Comment:
@rudolfix could you elaborate on this change?

Reply:
@burnash I'll fix this, thanks for pointing out